File: ResourceManager/ViewModels/Activities/ActivityPageViewModel.vb
Imports VBNetCore.Mvvm.Abstractions
Imports VBNetCore.Security
Imports ResourceManager.Configuration
Imports ResourceManager.Helpers
Imports System.Collections.ObjectModel
Imports System.Linq
Imports Microsoft.Extensions.DependencyInjection

Namespace ResourceManager.ViewModels
    Public Class ActivityPageViewModel
        Inherits PageViewModel
        Public Sub New()

            PageTitle = "Activity"
            PageIcon = AssetManager.Instance.GetImage("Activity.png")
            PageColor = ActivityPageColor
            MyBase.HasPermission = AuthManager.User.IsInPermission(VIEW_ACTIVITY_PERMISSION)

            Dim tagsContent = Application.ServiceProvider.GetRequiredService(Of TagsContentViewModel)()
            tagsContent.PageColor = PageColor
            tagsContent.ActivityColumnWidth = 110

            MenuItems = New ObservableCollection(Of PageContentViewModel) From {
    Application.ServiceProvider.GetRequiredService(Of ActivityContentViewModel)(),
    tagsContent
}

            SelectedItem = MenuItems.FirstOrDefault()
        End Sub
    End Class
End Namespace

File: ResourceManager/ViewModels/Activities/ActivityPageViewModel.cs
using Microsoft.Extensions.DependencyInjection;
using NetCore.Mvvm.Abstractions;
using NetCore.Security;
using ResourceManager.Configuration;
using ResourceManager.Helpers;
using System;
using System.Collections.ObjectModel;
using System.Linq;

namespace ResourceManager.ViewModels
{
    public class ActivityPageViewModel : PageViewModel
    {
        public ActivityPageViewModel()
        {

            PageTitle = "Activity";
            PageIcon = AssetManager.Instance.GetImage("Activity.png");
            PageColor = Constants.ActivityPageColor;
            HasPermission = AuthManager.User.IsInPermission(Constants.VIEW_ACTIVITY_PERMISSION);

            var tagsContent = App.ServiceProvider.GetRequiredService<TagsContentViewModel>();
            tagsContent.PageColor = PageColor;
            tagsContent.ActivityColumnWidth = 110;

            MenuItems = new ObservableCollection<PageContentViewModel>
            {
                App.ServiceProvider.GetRequiredService<ActivityContentViewModel>(),
                tagsContent
            };

            SelectedItem = MenuItems.FirstOrDefault();
        }
    }
}
